src/Pos/Wallet/Web/ClientTypes.hs

```diff
@@ -161,21 +161,22 @@ convertTxOutputs :: [TxOut] -> [(CId w, CCoin)]
 convertTxOutputs = map (addressToCId . txOutAddress &&& mkCCoin . txOutValue)
 
 -- | Get all addresses of source account of given transaction.
-getLocalAccountAddrs
-    :: [CWAddressMeta]  -- ^ all addresses in wallet
-    -> [CId Addr]       -- ^ Input addresses of transaction
-    -> Either Text [CId Addr]
-getLocalAccountAddrs walAddrMetas inputAddrs = do
+getChangeAccountAddrs
+    :: [CWAddressMeta]   -- ^ All addresses in wallet
+    -> [CId Addr]        -- ^ Input addresses of transaction
+    -> Either Text (Maybe [CId Addr])
+                         -- ^ `Just` change addrs if the wallet is source of
+                         --   transaction, `Nothing` otherwise
+getChangeAccountAddrs walAddrMetas inputAddrs = do
     someInputAddr <-
         head inputAddrs `whenNothing`
```

Comment:
This will cease working as soon as we support multiple input accounts for a single transaction. Can you register this as an issue in the bugtracker?